Tryouts/Core/Abstractions/ISubscriptionClient.cs

```diff
@@ -18,10 +18,13 @@ public interface ISubscriptionClient
     {
         Task Connect(string host, int port);
```

Comment:
App devs should not be concerned about configuration values like host and port. The client shouldn't even have a `Connect` method. This should be set up once on startup (module load) and the client implementation should open the connection when necessary.

Reply:
Eventually it should move over to a configuration module I agree. This is just a small prototype. I'll open another Jira to configure the end points.

Reply:
Here is a new github issue covering this: https://github.com/morganstanley/ComposeUI/issues/98